runtime/lua/vim/lsp/codelens.lua

```diff
@@ -53,9 +53,8 @@ local function execute_lens(lens, bufnr, client_id)
     return
   end
   client.request('workspace/executeCommand', command, function(...)
-    local result = vim.lsp.handlers['workspace/executeCommand'](...)
```

Comment:
Afaik handlers can have results, this doesn't look right?

Reply:
but this result never used  after this has `return result`